Updates based on onboarding feedback (#7272)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/career-coach.md

  
3. Select **Apply**.

## Making Career Coach available to your organization

> Now that Career Coach has been configured for your organization. Follow these steps to ensure that Career Coach is available to organization in Microsoft Teams.

Reviewer: ```suggestion
Now that Career Coach has been configured for your organization, follow these steps to ensure that Career Coach is available to your organization in Microsoft Teams.
```

  > [!NOTE]
  > Allowed means that the app is available for users in your educational institution. Blocked means that the app isn't available to students.

### Add Career Coach as an installed app

> [!NOTE]

> > This step ensures 1) that Career Coach is properly configured for your organization 2) that students find Career Coach.

Reviewer: ```suggestion
> This step ensures that Career Coach is properly configured for your organization and that students find Career Coach.
```

1. Sign in to the **Teams admin center**.

> 2. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*. 

Reviewer: ```suggestion
2. Select **Teams apps** &gt; **Setup policies** &gt; **Your policy**. 
```

> 3. Under Installed apps, select Add apps.

Reviewer: ```suggestion
3. Under Installed apps, select Add apps.
```

> 4. In the Add installed apps pane, search for the apps you want to automatically install for users when they start Teams. You can also filter apps by app permission policy. When you've chosen your list of apps, select Add.

Reviewer: ```suggestion
4. In the Add installed apps pane, search for the apps you want to automatically install for users when they start Teams. You can also filter apps by app permission policy. When you've chosen your list of apps, select **Add**.
```

### Pin the app

Pinning Career Coach will make the app more accessible and visible for students.

1. Sign in to the **Teams admin center**.

> 2. Select **Teams apps** &gt;**Setup policies** &gt; *Your policy*. 

Reviewer: ```suggestion
2. Select **Teams apps** &gt;**Setup policies** &gt; **Your policy**. 
```

3. Under **Pinned apps**, choose **Add apps**.

4. Search for **Career Coach**, and then select **Add**.

5. Choose the order for the app to appear and select **Save**.

> [!NOTE]
> Students will be notified in Microsoft Teams that Career Coach has been pinned.

> Reference [Manage app setup policies in Microsoft](https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies) for additional details.

Reviewer: ```suggestion
For more information, see [Manage app setup policies in Microsoft](/microsoftteams/teams-app-setup-policies) for additional details.
```
